product_to_bizerba_scale/views/product_scale_log_view.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<odoo>
+
+    <record id="view_product_scale_log_tree" model="ir.ui.view">
+        <field name="model">product.scale.log</field>
+        <field name="arch" type="xml">
+            <tree name="scale_log" string="Scale Log" decoration-muted="sent == True">
+                <field name="log_date"/>
+                <field name="sent"/>
+                <field name="last_send_date"/>
+                <field name="scale_system_id"/>
+                <field name="product_id"/>
+                <field name="action"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="view_product_scale_log_form" model="ir.ui.view">
+        <field name="model">product.scale.log</field>
+        <field name="arch" type="xml">
+            <form name="scale_log" string="Scale Log" version="7.0">
```

Comment:
Does the attribute `version` make sense now ?

Reply:
removed